backend/icpy/api/endpoints/hop.py

```
@@ -160,16 +160,26 @@ async def connect(payload: HopConnectRequest):
         logger.info(f"[HopAPI] Connect requested for credential={payload.credentialId} (asyncssh_available={ASYNCSSH_AVAILABLE})")
         session = await service.connect(payload.credentialId, password=payload.password, passphrase=payload.passphrase)
         logger.info(f"[HopAPI] Connect result: status={session.status} host={session.host} user={session.username} error={session.lastError}")
+        # Enrich payload with credentialName for friendly display on initial connect
+        payload_with_name = session.__dict__.copy()
+        try:
+            if getattr(session, 'credentialId', None):
+                cred = service.get_credential(session.credentialId)  # type: ignore[attr-defined]
+                if cred:
+                    payload_with_name['credentialName'] = cred.get('name')
+        except Exception:
+            pass
```

Ruff (0.14.0)

`try`-`except`-`pass` detected, consider logging the exception

(S110)